Formula/xplanet.rb

```diff
@@ -38,6 +38,11 @@ class Xplanet < Formula
   depends_on :x11 => :optional
 
   def install
+    # This is a simple fix to a bug in Xplanet 1.3.1 (-num_times=2 doesn't work):
+    # http://xplanet.sourceforge.net/FUDforum2/index.php?t=msg&th=671&start=0&S=07920715eb9a74361428ce957b99ae66
+    # This block should be removed with the next release of Xplanet
+    inreplace "src/libdisplay/DisplayOutput.cpp", "if (stopIndex > 1)", "if (stopIndex > 0)"
```

Comment:
Not sure if SourceForge allows for a raw patch download, so https://sourceforge.net/p/xplanet/code/208/tree//trunk/src/libdisplay/DisplayOutput.cpp?diff=5056482efd48f8457fc7910a:207 should probably just be added to the formula-patches repo.